Formula/rargs.rb

```diff
@@ -0,0 +1,16 @@
+class Rargs < Formula
+  desc "Util like xargs + awk with pattern matching support"
+  homepage "https://github.com/lotabout/rargs"
+  url "https://github.com/lotabout/rargs/archive/v0.2.2.tar.gz"
+  sha256 "ac6cf3a31ff5b55f86487fa3d3266edf8f562cc6b548d6e636daf373534388ad"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
+  end
+
+  test do
+    system "#{bin}/rargs", "-h"
```

Comment:
Is there a test that could be done to test the program works as expected?

Reply:
A simple test is added.